### DISCUSSION TITLE
SBE, Satlantic, WET Labs and Sea-Bird Scientific: how to match sensors to the correct manufacturer

### DISCUSSION THREAD

**Discussion by vpaba** [OneArgo/ArgoVocabs#28] [Q&A]
A question has been raised in the Argo community re: the matching of SUNA sensors to the correct manufacturer. This uncovered several issues which I tried to capture below.

For backround:
SUNA and SUNA_V2 are listed in the[ Argo metadata reference spreadsheet ](https://docs.google.com/spreadsheets/d/1Aw8B7FFUjG4e9MveD5qqvI7ZbB-z_x32IQnpHZbXVZA/edit#gid=6) as two separate spectrophotometers measuring nitrate. In it:
- the SUNA sensor is linked to MBARI as the sensor maker
- the SUNA_V2 sensor is linked to SATLANTIC as the sensor maker

However, it was specified that:
- the original SUNA sensor was developed at SATLANTIC (now Sea-Bird Scientific)
- the SUNA_V2 sensor is the (only?) one mounted on Argo floats - this sensor is also known as the Deep SUNA as it has a depth rating of 2000db. The first few SUNAs that were deployed were custom-built at SATLANTIC and are probably just listed as SUNA in the metadata.

The question then became whether the reference spreadsheet should be updated to:
- match the SUNA sensor with SATLANTIC
- match the SUNA_V2 sensor to SBE and/or SATLANTIC

From the '[About Sea-Bird Scientific](https://www.seabird.com/about-sea-bird)' webpage on the Sea-Bird Scientific website, it is stated that 'Sea-Bird Electronics was acquired by Danaher in 2008. Subsequent acquisition of WET Labs (2010) and Satlantic (2011) led the three companies to combine and form a new entity — Sea-Bird Scientific'.

These some of the issues that arise from this, of interest to the @nvs-vocabs/avtt:

1. The current information in the Argo metadata spreadsheet is incorrect as MBARI has never developed SUNA sensors.
2. There are currently two metadata entries being used in Argo NetCDF files (as well as the Argo metadata reference spreadhseet) which refer to the same sensor: SUNA and SUNA_V2. Which name should we retain? Should we update legacy metadata to correct this duplication?
3. The early SUNA sensors deployed on Argo floats were manufactured at SATLANTIC before the company's acquisition by Danaher and creation of Sea-Bird Scientific sometime after 2011. Therefore, depending on the timeline, the SUNA (aka SUNA_V2) sensor is linked to both SATLANTIC and Sea-Bird Scientific.
4. We currently don't know when exactly/from which serial numbers sensors manufactured at the three companies (Sea-Bird Electronics, WET Labs and Satlantic) officially changed manufacturer's brand following the merging into 'Sea-Bird Scientific' sometime after 2011. I have emailed Sea-Bird Scientific about this, and am awaiting reply.
5. In the [metadata reference spreadsheet for SENSOR_MAKER](https://docs.google.com/spreadsheets/d/1Aw8B7FFUjG4e9MveD5qqvI7ZbB-z_x32IQnpHZbXVZA/edit#gid=3), it doesn't seem to be recognised that Sea-Bird Electronics (SBE) also underwent a rebranding together with Satlantic and WET Labs. Should a new code be created for Sea-Bird Scientific (e.g. SEABIRD_SCIENTIFIC or just SEABIRD)? How would this affect all sensors currently being linked to SBE? A solution may be found in the way this code is moved to the NVS - the code could be kept as 'SBE', though the name and description could specify that this refers to Sea-Bird Scientific (though this solution may not be consisted with how the SATLANTIC and WETLABS codes are still currently used).
6. Should sensors originally manufactured at the three companies now be linked to Sea-Bird Scientific only for all new floats? Or should they be mapped to Sea-Bird Scientific as well as the original manufacturer (i.e. either SBE, Satlantic or WET Labs)? Or should this be decided on a case-by-case basis (i.e. depending on when the sensor was purchased)? Should we update legacy metadata to reflect the fact that Sea-Bird Scientific has been the official manufacturer since the three companies were merged?

Any thoughts welcome!

**Comment by matdon17** [comment]
@vpaba regarding no. 5 Sea-Bird Electronics and Sea-Bird Scientific, I think the existing entry in R26 for Sea-Bird is nearly sufficient to describe both entities, although it is currently perhaps a little deficient for the earlier Sea-Bird Electronics:

http://vocab.nerc.ac.uk/collection/R26/current/SBE/

Given the extensive used of 'SBE' in the data system I'm inclined to suggest continuing with the sole usage of this for both entities.

On no. 6 I think each individual sensor should be described by a single combination of SENSOR_MAKER and SENSOR_MODEL in the NetCDFs, and this should reflect the actual organisation that made that sensor, with (preferably) a clear and well-defined break between mergers/changes.  This doesn't stop the same sensor being linked to more than one manufacturer in the NVS through mappings.

**Reply by vpaba** [reply]
@matdon17 thanks for your replies.
On no. 5, I agree with you - I too think it is best to keep 'SBE' as is for the reason you mention. The description can be updated if needed.
On no. 6, that's right. On the NVS we can map to both, and I suggest we specify in the sensor model's description the date and/or serial number from which the official manufacturer changed from Sea-Bird Electronics/Satlantic/WET Labs to Sea-Bird Scientific, so that DACs know what sensor maker to use in the NetCDF. Hopefully Sea-Bird will provide us with this information.

**Comment by mbelbeoch** [comment]
there are 33 SUNA at OceanOPS:
so we can make the difference  SUNA/SATLANTIC  SUNA_V2/SBE  (so let see SBE reply)


[(https://www.ocean-ops.org/board?t=argo&contacts=%7B%22cruiseSampleCount%22%3A%220%22%2C%22query%22%3A%22def%22%2C%22docSampleCount%22%3A%220%22%2C%22themeId%22%3A%221000620%22%2C%22shipSampleCount%22%3A%220%22%2C%22obsSampleCount%22%3A%220%22%2C%22lineSampleCount%22%3A%220%22%2C%22contactSampleCount%22%3A%220%22%2C%22programSampleCount%22%3A%220%22%2C%22ptfSampleCount%22%3A%220%22%7D&cruises=%7B%22cruiseSampleCount%22%3A%220%22%2C%22docSampleCount%22%3A%220%22%2C%22themeId%22%3A%221000620%22%2C%22shipSampleCount%22%3A%220%22%2C%22obsSampleCount%22%3A%220%22%2C%22lineSampleCount%22%3A%220%22%2C%22contactSampleCount%22%3A%220%22%2C%22programSampleCount%22%3A%220%22%2C%22ptfSampleCount%22%3A%220%22%2C%22queryId%22%3A%22default%22%7D&ships=%7B%22ident%22%3A%22102mbelbeoch%22%2C%22cruiseSampleCount%22%3A%2252%22%2C%22query%22%3A%22def%22%2C%22docSampleCount%22%3A%220%22%2C%22themeId%22%3A%221000620%22%2C%22shipSampleCount%22%3A%220%22%2C%22obsSampleCount%22%3A%220%22%2C%22lineSampleCount%22%3A%220%22%2C%22contactSampleCount%22%3A%223%22%2C%22programSampleCount%22%3A%220%22%2C%22ptfSampleCount%22%3A%223884%22%2C%22token%22%3A%22clgrcOtdwCSpxQpKbjuwcgZb%22%7D&detailedKpiParams={"w":902,"h":727,"x":829,"y":297}&ptfTableColumns=%5B%22ref%22%2C%22status%22%2C%22model%22%2C%22country%22%2C%22programName%22%2C%22networks%22%2C%22deplDate%22%2C%22deplLat%22%2C%22deplLon%22%2C%22lastLocDate%22%2C%22serialNo%22%2C%22sensors%22%5D&qparams=%7B%22currentObsSqlQueryA%22%3A%22%20%22%2C%22beached%22%3A%220%22%2C%22allDacs%22%3A%220%22%2C%22allGdacs%22%3A%220%22%2C%22iced%22%3A%220%22%2C%22exactRefOnly%22%3A%22on%22%2C%22warningsChecked%22%3A%223%22%2C%22blacklist%22%3A%222%22%2C%22allNotGdacs%22%3A%221%22%2C%22sensormodels%22%3A%22itemid-2040%22%2C%22masterNetworks%22%3A%221000620%22%2C%22hasWigosId%22%3A%220%22%7D&modules=[{"id":1},{"id":2}])]

**Comment by vpaba** [comment]
I received an initial reply from Sea-Bird: "The separate branding for WETLabs, Satlantic, and Sea-bird Electronics continued well after 2011.  The actual transition to labelling all new sensors as "Sea-bird Scientific" may not have occurred at the same time across all product lines, and in my recollection may have extended as early as 2013 or late as 2016 for some products."

This means that we cannot use a blanket date across all sensors that underwent this rebranding. 
Hopefully we will receive more details - Sea-Bird are looking further into this.

**Reply by vpaba** [reply]
Update: Sea-Bird informed me that they could not provide any further information on this matter.

**Comment by delphinedobler** [comment]
Issue now solved. Sensor_models are mapped to their maker.
